app/services/scholars_archive/handle_error_logging_service.rb

```diff
@@ -0,0 +1,20 @@
+module ScholarsArchive
+  class HandleErrorLoggingService
+    def self.log_no_files_found_error(handle_params, work, handle_url)
```

Comment:
I feel like we should add a ticket related to this service to fire off an email to `ENV['SCHOLARSARCHIVE_ADMIN_EMAIL']` (or whatever the new ENV Greg just created) for these errors as well.

Reply:
https://github.com/osulp/Scholars-Archive/issues/881